Documentation.html

```diff
@@ -863,6 +863,47 @@ <h3>Example</h3>
       <pre class="code-example"><code>=WIKIDATAQID("en:Berlin")</code></pre>
       <pre class="code-result"><code>Q64</code></pre>
     </section>
+    <section>
+      <h2>WIKIDATALOOKUP</h2>
+      <p>Returns the Wikidata qid of the given identifier and property, see </p>
```

Comment:
This sentence seems to be incomplete. The `<p>` tag should be closed.

Reply:
Fixed, thanks.